.github/workflows/continuous-integration.yml

```diff
@@ -148,7 +149,7 @@ jobs:
           composer-options: "--ignore-platform-req=php+"
 
       - name: "Run PHPUnit"
-        run: "vendor/bin/phpunit -c ci/github/phpunit/oci8.xml --coverage-clover=coverage.xml"
+        run: "vendor/bin/phpunit -c ci/github/phpunit/oci8${{ matrix.oracle-version < 23 && '-21' || ''  }}.xml --coverage-clover=coverage.xml"
```

Comment:
needed because different dbname is needed - https://github.com/gvenzl/oci-oracle-xe/issues/238